docs/reference/operators/oci.md

```diff
@@ -0,0 +1,82 @@
+---
+title: oci
+sidebar_position: 0
+---
+
+The OCI operator handles Gadgets images. It pulls images from the OCI registries
+and invokes the different layer operators according to the
+[layers](../../spec/oci.md#image-layers-and-media-types) present on the image.
+
+## Global Parameters
+
+### `verify-image`
+
+Verify image using the provided public key. Check [Verify image-based
+gadgets](../verify-assets.mdx#verify-image-based-gadgets) to learn more.
+
+Default: `true`
+
+### `public-keys`
+
+Public keys used to verify the gadgets. Check [Verify image-based
+gadgets](../verify-assets.mdx#verify-image-based-gadgets) to learn more.
+
+Default: [Inspektor Gadget public
+key](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/resources/inspektor-gadget.pub).
+
+### `allowed-gadgets`
+
+List of allowed gadgets. If a gadget is not part of it, execution will be
+denied. By default, all digests are allowed. Check [Restricting
+Gadgets](../restricting-gadgets.mdx) to get more details.
+
+### `insecure-registries`
+
+List of registries to access over plain HTTP. Check [Insecure
+Registries](../insecure-registries.mdx) to learn more.
+
+### `disallow-pulling`
+
+Disallow pulling gadgets from registries. Check [Disallow pulling
+Gadgets](../disallow-pulling.mdx) to learn more.
+
+Default: `false`
+
+## Instance Parameters
+
+### `authfile`
+
+TODO: is this really a instance param?
+
+Path of the authentication file. This overrides the `REGISTRY_AUTH_FILE`
+environment variable.
```

Comment:
When using client/server (gadgetctl or kubectl-gadget), is this the environment variable on the client or server process?

Is the path referring to a file on the client machine or the server machine?

Reply:
It's the path on the server. According to the TODO above, this shouldn't the exposed when running gadgetctl or kubectl-gadget. I'll handle that in another PR.